sbnanaobj/StandardRecord/SROpFlash.h

```diff
@@ -0,0 +1,48 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SROPFLASH.h
+// \brief   Adaptation of recob::OpFlash (https://nusoft.fnal.gov/larsoft/doxsvn/html/classrecob_1_1OpFlash.html). This SR code copied/based on other SR objects.
+// \author  jsmedley@fnal.gov
+////////////////////////////////////////////////////////////////////////
+#ifndef SROPFLASH_H
+#define SROPFLASH_H
+
+#include "sbnanaobj/StandardRecord/SRConstants.h"
+#include <climits>
+#include <vector>
+
+namespace caf
+{
+  /// OpFlash
```

Comment:
At least put `Adaptation of recob::OpFlash` here. Maybe include a bit of human description, something like "Optical Flash -- a summary of multiple optical hits that have been determined to be associated"